clouddriver-eureka/src/main/groovy/com/netflix/spinnaker/clouddriver/eureka/deploy/ops/AbstractEurekaSupport.groovy

```diff
@@ -146,8 +146,12 @@ abstract class AbstractEurekaSupport {
           }
         }
       } catch (RetrofitError retrofitError) {
-        if (retrofitError.response?.status == 404 && discoveryStatus == DiscoveryStatus.Disable) {
-          task.updateStatus phaseName, "Could not find ${instanceId} in application $applicationName in discovery, skipping disable operation."
+        if (discoveryStatus == DiscoveryStatus.Disable) {
```

Comment:
does this get called when I take a single instance out of discovery?
if so... maybe we should also check that `instanceIds.size > 1`?

Reply:
updated in 4a7237bf87f858a890a1a1096fbbb4b023bacae9, instead of checking `instanceIds.size` I pass a `strict=true` flag when we disable one or more instances explicitly but `strict=false` when disabling a server group. What do you think?